nixos/modules/services/networking/nat.nix

```diff
@@ -50,7 +50,7 @@ let
     # NAT from external ports to internal ports.
     ${concatMapStrings (fwd: ''
       iptables -w -t nat -A nixos-nat-pre \
-        -i ${cfg.externalInterface} -p ${fwd.proto} \
+        -i ${toString cfg.externalInterface} -p ${fwd.proto} \
```

Comment:
Does a rule with `-i ""` actually do what we want, i.e. match any interface ?  `iptables` manpage doesn't comment on that case.

Reply:
I am not sure that matching on all interfaces would work for ```forwardPorts``` or if it has any practical sense.
May be worth to check.
Anyway it is protected by ```assertions``` below

Reply:
That might be useful for Dynamic DNS case